app/src/main/java/de/danoeh/antennapod/activity/PreferenceActivity.java

```diff
@@ -83,6 +94,7 @@ private PreferenceFragmentCompat getPreferenceScreen(int screen) {
         return prefFragment;
     }
 
+    @SuppressLint("NonConstantResourceId")
```

Comment:
Please don't suppress this. It's an actual problem that should be fixed at some point (not the scope of this PR and I don't like PRs that change dozens of unrelated files at once - so it should be changed when touching those lines for another reason in the future)